File: src/main/java/no/nav/pensjon/innsyn/domain/desc/OmsorgDescriptor.java
package no.nav.pensjon.innsyn.domain.desc;

import no.nav.pensjon.innsyn.entity.EntityDescriptor;

public class OmsorgDescriptor implements EntityDescriptor {

    private static final String ENTITY_NAME = "Omsorg";

    private static final String[] PROPERTY_NAMES = {
            "År",
            "Kilde",
            "Type",
            "Status"
    };

    public String getEntityName() {
        return ENTITY_NAME;
    }

    public String[] getPropertyNames() {
        return PROPERTY_NAMES;
    }
}

File: src/main/kotlin/no/nav/pensjon/innsyn/domain/desc/OmsorgDescriptor.kt
package no.nav.pensjon.innsyn.domain.desc

import no.nav.pensjon.innsyn.domain.Omsorg
import no.nav.pensjon.innsyn.entity.EntityDescriptor

object OmsorgDescriptor : EntityDescriptor<Omsorg>("Omsorg",
        arrayOf(
                "År",
                "Kilde",
                "Type",
                "Status"
        )
)